client/oauth2.go

```diff
@@ -8,8 +8,8 @@ import (
 
 const (
 	// generate using many fair dice rolls
-	OAUTH2_CLIENT_ID     = "d0f032e5af41187fdaf45b4aeee76ee4"
-	OAUTH2_CLIENT_SECRET = "4141f52bb9f80dfc776b5b773ccf550d"
+	Oauth2ClientID     = "d0f032e5af41187fdaf45b4aeee76ee4"
+	Oauth2ClientSecret = "4141f52bb9f80dfc776b5b773ccf550d"
```

Comment:
I think the correct capitalization would be `OAuth2ClientFoo`?

Reply:
Yes in these cases I think it should be `OAuth2ClientFoo`. I shall change.